<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Project 3 Page</title>
    <link rel="stylesheet" href="Resources\style.css">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Mulish&family=Roboto&display=swap" rel="stylesheet">

</head>
<body>

    <header>

        <h1 id="Project_header">Genome Assembly</h1>

    </header>

    <p>
        My junior year at the University of Kentucky I was fortunate enough to work with Dr. Mark L. Farman
        and a team of a mix of biology and computer science majors to assemble different strains of a fungus
        <em>Pyricularia Oryzae's</em> genome that was infecting different types of host plants (mainly grains and grasses).
        During this process I was able to get hands on experience working with very large sets of data in a 
        coordinated team environment, processing billions of nucleotides worth of data. To do this, our team 
        utilized the University of Kentucky's super computer, linux OS, github, DNA assembly software, scripting,
        on top of clear and concise communication.   
    </p>
 
    <img src="Resources/GrayLeafSpot.jpg" alt="photo from https://ag.umass.edu/turf/fact-sheets/gray-leaf-spot-of-ryegrass-tall-fescue">

    <p>
        <em>Pyricularia Oryzae</em> is a species of fungus that infects grasses and grain plants causing a host of 
        different diseases such as rice blast, wheat blast, and grey leaf spot. The fungal genome I was 
        assembling was found on an <em>Festuca Arundinacea</em> tall grass in Uruguay. Each team member had 
        to assemble a <em>Pyricularia Oryzae</em> genome sourced from different host plants that the fungus was infecting.
        The goal of this was so that once we have assembled the sequenced data (our fungal genomes), we would compare
        and contrast the data in hopes to gain insights on the origins of this fungus, what genes were common among
        certain host species, and even fungal evolution as a whole. Our work would go on to support Dr. Farman's research
        which he scheduled to publish our findings in December of 2024. 
    </p>

    <p>
        If you would like to know more about the process of how we did this, check out my github page 
        <a href="https://github.com/gavinRobinson/MyGenome/blob/main/README.md">here</a>
    </p>

    <a href="./index.html">Back</a>

</body>